### PR TITLE
Add istio scrap targets to VMI system prometheus

### PR DESCRIPTION
**Pull request by bobdonat**
This pull request adds Istio scrap targets to the VMI system Prometheus.

### CODE REVIEW

File: pkg/resources/helper.go [hunk 1]
```diff
@@ -279,7 +283,44 @@ scrape_configs:
      target_label: kubernetes_namespace
    - source_labels: [__meta_kubernetes_pod_name]
      action: replace
-     target_label: kubernetes_pod_name`)
+     target_label: kubernetes_pod_name
+
+ # Scrape config for Istio envoy stats
```
Comment (pfmackin) [inline pkg/resources/helper.go:288]: Curious if Istio aggregates envoy metrics for all the envoy sidecars and the aggregation is what is being scraped.

**Review by pfmackin — APPROVED** [review]

File: pkg/resources/helper.go (hunk 1 above)
Comment (bobdonat) [reply pkg/resources/helper.go:288]: envoy metrics are per pod

**Review by pfmackin — CHANGES_REQUESTED** [review]
Just noticed copyright needs to be changed

**Comment by bobdonat** [comment]
> Just noticed copyright needs to be changed

Fixed.

**Review by pfmackin — APPROVED** [review]

**Pull request merged**